docs/api_static/script.js

```diff
@@ -0,0 +1,18 @@
+"use strict";
+
+function addLink(element) {
+  const a = document.createElement("a");
```

Comment:
change const to var is not too much work, let's stick to ES5 for a while(at least for enterprise windows 7 users, they have to use IE sometimes).
If you want dogfood bucklescript inside this repo, see one example: https://github.com/bloomberg/bucklescript/blob/master/scripts/clean.ml
https://github.com/bloomberg/bucklescript/blob/master/scripts/clean.js